### PR TITLE
Adding AWS RDS reference app

### PR DESCRIPTION
**Pull request by willdavsmith**
* Added AWS RDS reference app

### CODE REVIEW

File: docs/content/getting-started/reference-apps/aws-rds/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Prerequisites
+
+- [Complete the getting started guide for AWS up to Step 2]({{< ref aws-quickstart >}})
+
+## Step 1: Create a Bicep file which uses AWS RDS
+
+This Bicep file defines:
+* A `container` resource which serves the WordPress application.
+* An RDS instance of a MySQL database, along with a subnet group resource. The setup here ensures that the RDS instance is deployed to the same subnet group and VPC as your EKS cluster, and additionally has the correct security group to enable communication between pods in the EKS cluster and the database.
```
Comment (jkotalik) [inline docs/content/getting-started/reference-apps/aws-rds/index.md]: Separate RDS and Subnet into different bullets

**Review by jkotalik — DISMISSED** [review]

File: docs/content/getting-started/reference-apps/_index.md [hunk 2]
```diff
@@ -36,6 +36,13 @@ no_list: true
   </a>
 </div>
 {{% /card %}}
+{{% card header="[WordPress on AWS](./aws-rds)" footer="Learn about how to model and deploy a WordPress site using AWS RDS with Project Radius" %}}
+<div class="text-center">
+  <a href="./aws-rds">
+    <img src="./aws-rds/aws-rds.png" alt="AWS RDS Logo">
+  </a>
+</div>
+{{% /card %}}
```
Comment (AaronCrawfis) [inline docs/content/getting-started/reference-apps/_index.md]: These cards now auto-generated (just upmerged this change into edge from v0.16). You can remove this change.

Also please rename `aws-rds.png` to `icon.png`

File: docs/content/getting-started/reference-apps/aws-rds/snippets/app.bicep [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+param subnetGroupName string
+resource subnetGroup 'AWS.RDS/DBSubnetGroup@default' = {
+  properties: {
+    DBSubnetGroupName: subnetGroupName
+    DBSubnetGroupDescription: subnetGroupName
+    SubnetIds: eksCluster.properties.ResourcesVpcConfig.SubnetIds
+  }
+}
+
+param databaseIdentifier string
```
Comment (AaronCrawfis) [inline docs/content/getting-started/reference-apps/aws-rds/snippets/app.bicep]: For these params could you add an @description block that provides info on how to pick a name? Either a link to an AWS docs page or inline details would be good. Specifically if this needs to be globally unique or account unique, what are the character/naming restrictions, and other considerations

Comment (AaronCrawfis) [reply docs/content/getting-started/reference-apps/aws-rds/snippets/app.bicep]: Could we also move them up to the top so all the params are together? Usually the convention is to put all params, especially required params, at the top of a Bicep file

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: docs/content/getting-started/reference-apps/aws-rds/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    rad deploy ./app.bicep -p eksClusterName=<YOUR_EXISTING_EKS_CLUSTER_NAME> -p subnetGroupName=<YOUR_SUBNET_GROUP_NAME> -p databaseIdentifier=<YOUR_DATABASE_NAME> -p databasePassword=<YOUR_DATABASE_PASSWORD>
+    ```
+
+1. Port-forward the container to your machine with [`rad resource expose`]({{< ref rad_resource_expose >}}):
+
+    ```bash
+    rad resource expose containers wordpress-container --application wordpress-app --port 8080 --remote-port 80
+    ```
+
+   Note: the `--remote-port` flag is set because the port exposed by the `wordpress` image is `80`.
```
Comment (AaronCrawfis) [inline docs/content/getting-started/reference-apps/aws-rds/index.md:43]: Is there an issue with port forwarding to 80?

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**